lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -81,18 +81,18 @@
 // Browse menu
 // -----------
 .forum-nav-browse-menu-wrapper {
-  border-bottom: 1px solid $gray-l3;
+  border-bottom: 1px solid $forum-color-border;
   background: $gray-l5;
 }
 
 .forum-nav-browse-filter {
   position: relative;
-  border-bottom: 1px solid $gray-l2;
+  border-bottom: 1px solid $forum-color-border;
   padding: ($baseline/4);
 }
 
 .forum-nav-browse-filter .icon {
-  @include font-size(12);
+  font-size: 12px;
```

Comment:
The ``font-size`` mixin wasn't originally available so I switched a lot of the lines over. I then decided it would be simpler to put a shim mixin in just to make it work, so I can revert this if you like. I don't really understand the benefit of doing it via a mixin... do you think it was a browser support relic?